openff/interchange/interop/gromacs/export/_export.py

```diff
@@ -60,31 +64,100 @@ def _write_defaults(self, top):
             f"{self.system.coul_14:8.6f}\n\n",
         )
 
-    def _write_atomtypes(self, top):
+    def _write_atomtypes(self, top, merge_atom_types: bool) -> dict[str, str]:
```

Comment:
It's great to see work toward the "atom type explosion" problem that's plagued us for years. But given my recollection of the severity of the parmed bug that arose from seemingly innocuous atom type merging, it may make sense to have this be `_merge_atom_types`(with the leading underscore) or require `INTERCHANGE_EXPERIMENTAL=1` for a few releases to communicate that it's new/experimental. Once early adopters have run with it for a while we can remove the underscore and make it a public method.

Reply:
That is fair, I added underscore to all public methods